libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/IpvSessionService.java

```diff
@@ -101,14 +100,14 @@ public IpvSessionItem generateIpvSession(
 
         ipvSessionItem.setCreationDateTime(Instant.now().toString());
 
-        ipvSessionItem.setUserState(generateStartingState(errorObject));
-
         ipvSessionItem.setVot(Vot.P0);
 
-        String journeyType = this.configService.getSsmParameter(ConfigurationVariable.JOURNEY_TYPE);
-        ipvSessionItem.setJourneyType(IpvJourneyTypes.valueOf(journeyType));
-
-        if (errorObject != null) {
+        if (errorObject == null) {
+            ipvSessionItem.setJourneyType(IpvJourneyTypes.INITIAL_JOURNEY_SELECTION);
+            ipvSessionItem.setUserState(START_STATE);
+        } else {
+            ipvSessionItem.setJourneyType(IpvJourneyTypes.TECHNICAL_ERROR);
+            ipvSessionItem.setUserState(ERROR_STATE);
```

Comment:
This means the initial journey is no longer controlled via config, but I'm not convinced we ever used that (or would have wanted to)